templates/about/release-cycle.html

```diff
@@ -235,11 +235,6 @@ <h2>Maintenance and security updates</h2>
     <div class="row">
       <div class="col-8">
         <p>Ubuntu LTS releases transition into Extended Security Maintenance (ESM) phase as the standard, five-year public support window comes to a close. It is recommended for users and organisations to upgrade to the latest LTS release or <a href="/esm">subscribe to ESM</a> for continued security coverage.</p>
-        <p>This command will print the exact status of your system:</p>
-        <div class="p-code-copyable" style="margin-top: .5rem;">
-          <input class="p-code-copyable__input" value="ubuntu-support-status" readonly="readonly">
```

Comment:
Looks like `ubuntu-security-status` is a successor of `ubuntu-support-status` at least on Ubuntu 20.04 LTS as of today. Instead of deleting the section, we could have conditions to use `ubuntu-security-status` > 20.04, then the previous command as `ubuntu-support-status` for older releases.

Reply:
I don’t know how to implement conditions, and I don’t think doing it with UA sniffing is a good idea, privacy-wise. Maybe tabs?